web/web/src/lib/utils/initial.js

```diff
@@ -48,6 +48,34 @@ export const messagingProviders = [
 ]
 
 export const providers = [
+  {
+    label: 'Apache Doris',
+    value: 'jdbc-doris',
+    defaultProps: [
+      {
+        key: 'jdbc-driver',
+        value: '',
```

Comment:
Is there are default value for this driver?

Reply:
yes